### PR TITLE
Fix css in grid

### PR DESCRIPTION
**Pull request by trikimahdi**

### CODE REVIEW

File: src/components/directory-content.js [hunk 1]
```diff
@@ -337,23 +337,25 @@ const DirectoryContent = () => {
...
-        return <Typography>{elementTypeLabel}</Typography>;
+        return (
+            <OverflowableText
+                text={elementTypeLabel}
+                tooltipStyle={classes.tooltip}
+            />
+        );
     }
 
     function typeCellRender(cellData) {
```
Comment (ghelloumi) [inline src/components/directory-content.js:346]: suggestion: 
```
function typeCellRender(cellData) {
        const {rowData = {}} = cellData || {}
        const elementUuid = rowData['elementUuid'];
        const objectType = rowData[cellData.dataKey];
        const {subtype, format} = childrenMetadata[elementUuid] || {}

        return (
            <div className={classes.cell}>
                {childrenMetadata[elementUuid] &&
                    getElementTypeTranslation(
                        objectType,
                        subtype,
                        format
                    )}
            </div>
        );
}
```

**Review by ghelloumi — COMMENTED** [review]
Can you change Copyright (c) 2021 to Copyright (c) 2023 please
thank you

**Review by ghelloumi — COMMENTED** [review]
- there is a multiple use of React.useState .. can you replace them with useState as useState is already imported.
I think that we are able to take advantage of this PR to improve some code

thank you

**Comment by trikimahdi** [comment]
> Can you change Copyright (c) 2021 to Copyright (c) 2023 please thank you

**Comment by trikimahdi** [comment]
@ghelloumi We don't change the year of licence (confirmed by @dbraquart )
Replacing React.useState with useState is not in this PR scope. It's a good idea but we can manage to create another PR for this.

File: src/components/directory-content.js (hunk 1 above)
Comment (trikimahdi) [reply src/components/directory-content.js:346]: done

**Review by ghelloumi — APPROVED** [review]

**Review by Walid-Sahnoun — APPROVED** [review]
code ok
test ok

**Pull request merged**